src/common/oclengine.cpp

```diff
@@ -54,39 +43,42 @@ OCLEngine& OCLEngine::operator=(OCLEngine const& rhs) { return *this; }
 
 void OCLEngine::InitOCL(int plat, int dev)
 {
+    int i;
     // get all platforms (drivers), e.g. NVIDIA
 
+    std::vector<cl::Platform> all_platforms;
+    std::vector<cl::Device> all_devices;
+    cl::Platform default_platform;
+    cl::Device default_device;
+
     cl::Platform::get(&all_platforms);
 
     if (all_platforms.size() == 0) {
         std::cout << " No platforms found. Check OpenCL installation!\n";
         exit(1);
     }
     default_platform = all_platforms[plat];
-    std::cout << "Using platform: " << default_platform.getInfo<CL_PLATFORM_NAME>() << "\n";
 
     // get default device (CPUs, GPUs) of the default platform
```

Comment:
Update comment?

Reply:
This isn't the head? It already has been updated, on the head.

Reply:
Oh, sorry. The comment. I thought you meant the std::cout. Done.